about.html

```diff
@@ -0,0 +1,118 @@
+      <li class="navbarListItem"><a class="navbarLink">Program</a></li>
+      <li class="navbarListItem"><a class="navbarLink">Join</a></li>
+      <li class="navbarListItem"><a class="navbarLink">Sponsor</a></li>
+      <li class="navbarListItem"><a class="navbarLink">News</a></li>
+      <li class="navbarListItem lastItem"><a class="navbarLink">Spring Festival</a></li>
+    </ul>
+  </nav>
+  <main>
+    <section class="headlineAbout">
+      <div class="headlineDivAbout">
+        <p class="smallTitle">"Hello! Music world"</p>
+        <h1 class="pageTitleAbout">CORDOBA CLASICAL<br/> SPRING FESTIVAL <br class="breakMobile"/>2023</h1>
+        <p class="aboutDescription">The Global Summit brings together the community of experts, academics, and activist who comprise
+          the Creative Commons affiliate networks in a different country every two.
+          This year, we hope to expand our invitations list including organizations
+          and individuals
+        </p>
+        <p class="aboutDescriptionDesktop">
+          Experience the magic of classical music at the Cordoba Spring Festival, as we celebrate
+           the brilliance of Beethoven, Mozart, and Handel. From powerful symphonies to exquisite concertos, 
+           immerse yourself in the enchanting world of these iconic composers. Join us for a series of captivating 
+           performances by talented musicians in the vibrant ambiance of Cordoba, 
+          creating unforgettable musical moments that resonate with the spirit of spring.
+        </p>
+        <p class="conctactInfo">Please contact us per Email for any further questions <br class="breakMobile"/>
+        Coroba Clasical Spring Festival</p>
+        <a class="email">cordobaclasicalfestival@gmail.cordoba.gob</a>
+      </div>
+    </section>
+    <section class="logoInfo" id="sectionLink1">
+      <div class="logoDescription">
+        <h2 class="logoDescriptionTitle">Cordoba<br class="breakMobile"/> Clasical Spring Festival Logo</h2>
+        <p class="logoDescriptionText">The logo of Cordoba Clasical Spring Festival was decided <br/>
+          through the logo competition from 8, june to 7, july.
+        </p>
+        <p class="logoDescriptionTextDesktop">The logo of Cordoba Clasical Spring Festivalwas decided through the logo
+          competition from 8.June to 7.July.<br/>After online public voting and Selection Comittee's judging 
+          process, the logo from Naresh Agrwal won the prize. <br/>After consultation with Mr.Agrawal we optimized 
+          the prize-wining-logo for this year's summit as following:
+        </p>
+      </div>
+      <div class="logoDiv">
+        <img src="./assets/navbar/orchestraIcon.jpg" class="logoInfoLogo" alt="Cordoba Spring Festival 2023 Icon">
+      </div>
+    </section>
+    <section class="pastConcerts" id="sectionLink2">
+      <h2 class="pastConcertTitle">See the past Cordoba Clasical Spring Festivals</h2>
```

Comment:
- [ ] Kindly add the red line under the title as in the provided design(see image)
![Screenshot from 2023-06-29 02-33-29](https://github.com/IsmaelMastronardi/Capstone1/assets/91142432/fe85b13a-7531-4cc6-8b75-b53c512f0db7)